doc/wallet-passphrase.md

```diff
@@ -0,0 +1,21 @@
+## Wallet Passphrase
+
+Understanding the nuances of wallet security is essential to storing your Bitcoin safely. One crucial aspect of safeguarding funds is the utilization of a wallet passphrase or password, implemented through wallet encryption. Let's explore key points to demystify the nature of a wallet passphrase and the encryption process, emphasizing what they do and what they don't do:
+
+  - **Not the Seed:**
+The wallet passphrase is distinct from the seed. The seed, also known as the HD seed, functions as a master key for deriving key pairs in a hierarchical deterministic (HD) wallet. The passphrase on the other hand, serves as an additional layer of security specifically designed to secure the private keys within the wallet. While the HD seed is essential for deriving private and public keys in the wallet, the passphrase serves as a safeguard, demanding an additional layer of authentication to access funds on the wallet. This dual-layered approach enhances overall wallet security, effectively mitigating risks associated with unauthorized access and potential theft.
+
+- **Limited Protection Against Physical Threats:**
```

Comment:
I don't think this is very relevant, even though it was suggested in the original issue.

Reply:
removed